src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -47,37 +65,34 @@ const styles = StyleSheet.create({
     height: 48,
     backgroundColor: colors.white
   },
-  tabItem: (index) => ({
-    flex: 1,
-    width: width / 4,
+  tabItem: (tabs) => ({
+    width: tabs === 4 ? undefined : windowWidth / tabs,
     justifyContent: 'center',
+    alignItems: 'center',
     height: '100%',
-    paddingLeft: index !== 1 ? 20 : 0
+    paddingHorizontal: tabs === 4 ? 20 : undefined
   }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `tabItem` style is using a parameter named `tabs`, which seems to be the number of tabs. It would be clearer to rename this parameter to something like `numTabs` or `tabCount`.
```diff
-  tabItem: (tabs) => ({
-    width: tabs === 4 ? undefined : windowWidth / tabs,
-    justifyContent: 'center',
-    alignItems: 'center',
-    height: '100%',
-    paddingHorizontal: tabs === 4 ? 20 : undefined
-  }),

+  tabItem: (numTabs) => ({
+    width: numTabs === 4 ? undefined : windowWidth / numTabs,
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '100%',
+    paddingHorizontal: numTabs === 4 ? 20 : undefined
+  }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->